src/app/swap/[[...path]]/page.tsx

```diff
@@ -1,13 +1,25 @@
 'use client'
 
 import { Flex } from '@chakra-ui/react'
+import { useSearchParams } from 'next/navigation'
+import { useMemo } from 'react'
 
+import {
+  ExodusCampaignBanner,
+  isExodusCampaign,
+} from '@/components/banners/exodus-campaign-banner'
 import { FliMigrationBanner } from '@/components/banners/fli-migration-banner'
 import { Swap } from '@/components/swap'
 import { useSelectedToken } from '@/lib/providers/selected-token-provider'
 
 export default function SwapPage() {
   const { inputToken, isMinting, outputToken } = useSelectedToken()
+  const searchParams = useSearchParams()
+  const showExodusCampaignBanner = useMemo(
+    () => isExodusCampaign(searchParams),
+    [searchParams],
+  )
```

Comment:
Maybe comment this out and just make it `false´ for now? So we can merge all this already.

Reply:
@janndriessen added a feature flag to disable for now/allow visual QA